scripts/deploy-docs.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+source hugo-check.sh
```

Comment:
Would it be potentially easier to set up hugo to install with npm using https://www.npmjs.com/package/hugo-installer?

Reply:
There's also a hugo-cli that might be useful https://www.npmjs.com/package/hugo-cli

Reply:
Yes that is awesome!!!

Reply:
I won't have time to add those packages this sprint, so I made #352 to keep track of it.  I'd like to merge this PR as-is and add those next sprint if no one objects.

Reply:
👍